in_toto/verifylib.py

```
@@ -1465,18 +1450,22 @@ def in_toto_verify(layout, layout_key_dict, link_dir_path=".",
     Runs inspection commands in subprocess.
 
   Returns:
-    A Metablock object that contains a Link object, which summarizes the
-    materials and products of the overall software supply chain. This is mostly
-    useful during recursive sublayout verification.
+    A Link object, which summarizes the materials and products of the overall
+    software supply chain. This is mostly useful during recursive sublayout
+    verification.
 
   """
-  LOG.info("Verifying layout signatures...")
-  verify_layout_signatures(layout, layout_key_dict)
+
+  keys = create_key_list_from_key_dict(layout_key_dict)
```

Same comment as above in `in_toto_sign.py` applies here. By silently dropping keys in `create_key_list_from_key_dict`, the expected threshold can be subverted without noticing.

---

I guess raising exception or logging it might help.